src/components/styled/StyledNavButton.js

```diff
@@ -12,3 +12,43 @@ export const StyledNavButton = styled.button`
 	font-size: 1em;
 	text-decoration: none;
 `;
+
+export const StyledHomeButton = styled.button`
+	display: flex;
+	flex-grow: 1;
+	align-items: center;
+	justify-content: center;
+	width: 100px;
+	margin-top: 60%;
+	margin-left: 38%;
+	padding: 12px 18px;
+	border: none;
+	border-radius: 2px;
+	outline: none;
+	background-color: #2196f3;
+	box-shadow: 0 0 4px #999;
+	color: white;
+	font-size: 16px;
+	text-transform: uppercase;
+	cursor: pointer;
+`;
+
+export const StyledBlackButton = styled.button`
+	margin: 6px 3px 6px 3px;
+	margin-bottom: 12px;
+	padding: 8px 20px;
+	border-radius: 5px;
+	background-color: black;
+	color: white;
+	font-size: 12px;
+`;
+
+export const StyledWhiteButton = styled.button`
+	margin: 6px 3px 6px 3px;
+	margin-bottom: 12px;
+	padding: 8px 20px;
+	border-radius: 5px;
+	background-color: white;
+	color: black;
+	font-size: 12px;
+`;
```

Comment:
Do you even use all these buttons? 😅 If not, please remove them from your code.

Reply:
Thanks, I were just using 2 of 4 Buttons.